src/Search.py

```diff
@@ -8,8 +8,8 @@
 
 class Search:
     def __init__(self, api, heuristic: AbstractHeuristic):
-        self.api = api
-        self.heuristic = heuristic
+        self.api = api()
```

Comment:
These two changes will probably break the WikiRacer.py code

Reply:
We should be passing in instances of the API and heuristic to Search.py, not the classes.